src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,55 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $watch-size;
+  width: $watch-size;
+  border-radius: 50%;
+  border: 1px solid $color-black;
+  background: url(https://i.pinimg.com/736x/4d/f1/c3/4df1c3d401264f15701ff473d22e4f92.jpg) center cover;
+
+  &__spot {
+    height: $spot-size;
+    width: $spot-size;
+    border-radius: 50%;
+    background-color: $color-black;
+  }
+
+  &__hand {
+    width: $hand-width;
+    border-radius: 10px;
+    position: absolute;
+    bottom: 50%;
+    transform-origin: bottom center;
+  
+    &--small {
+      height: 100px;
+      background-color: #00f;
+      animation: circle 60s steps(60) infinite;
+
+      @keyframes circle {
+        0% {
+          transform: rotate(0deg);
+        }
+        100% {
+          transform: rotate(360deg);
+        }
+      }
+    }
+    
+    &__hand--big {
```

Comment:
Is it a hand modifier? 
Move it into `&__hand` selector the same as you did with &--small